benchmarks/python/test_rope.py

```diff
@@ -176,3 +62,80 @@ def test_rope_benchmark(
 
     if not disable_benchmarking:
         run_benchmark(benchmark, fd.execute, inputs)
+
+
+@pytest.mark.parametrize(
+    "rope_variation",
+    [
+        "llama_2_7b_hf_rope",
+        "llama_3_8B_rope",
+        "hf_qwen2_rope",
+        "hf_phi3_rope",
+        "hf_mistral_nemo_rope",
+    ],
+)
+@pytest.mark.parametrize(
+    "executor", ["eager", "torchcompile", "thunder", "thunder-torchcompile"]
+)
+def test_rope_variations_fwd_benchmark(
```

Comment:
```suggestion
def test_rope_fwd_benchmark(
```
I don't see variations here add useful information. The parameter rope_variation clearly indicates that we are testing more than one variations.